lib/gateway/index.js

```diff
@@ -49,11 +44,19 @@ module.exports = function ({ plugins, config } = {}) {
     });
 };
 
-function bootstrap({ plugins, config } = {}) {
-  const app = express();
-  app.set('x-powered-by', false);
+const bootstrapPolicies = ({ app, plugins, config } = {}) => {
+  if (plugins && plugins.policies && plugins.policies.length) {
+    plugins.policies.forEach(policy => {
+      log.debug(`registering policy ${policy.name}`);
+      policies.register(policy);
+    });
+  }
+
+  // Load policies present in config
+  if (config.gatewayConfig.policies) {
```

Comment:
This field is required in the JSON Schema, so there's no chance it will be undefined. You can remove this check.